.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: Feature Request
+assignees: ''
+
+---
+
+<!--
+  - Use this issue template to request a feature in the deCONZ REST-API.
+  - If you want to request a feature for the Phoscon App, please head over to: https://github.com/dresden-elektronik/phoscon-app-beta
+  - If you're unsure if the request fits into this issue tracker, please ask for advise in our Discord chat: https://discord.gg/QFhTxqN
+-->
+
+## Feature request type
+<!--
```

Comment:
Also here refer to the other systems wikipage and that this github is only for feature requests to Deconz rest api

Reply:
> Instead of screenshots could we get a "dump" function that generates a text file that can be provided?

Brilliant!  That could actually be the first application of the new `DeRestPluginPrivate::Node` class: collect all the info for a dump.

Reply:
Awesome!